runner.js

```diff
@@ -486,6 +494,8 @@ main().catch(e => {
     msg = e.stack
   }
 
-  process.stderr.write(`${ chalk.bgRed(' ERROR ') } ${ chalk.red(msg) }\n`)
+  reporter.error({
+    message: `${ chalk.bgRed(' ERROR ') } ${ chalk.red(msg) }\n`
```

Comment:
Should we move ` ERROR ` to a reporter?

Reply:
Yeah, I think so. Also in `warn` method?

Reply:
Yeap

Reply:
Done for `warn()` and `error()` in `HumanReporter`